src/attack.c

```diff
@@ -340,6 +341,10 @@ launch_plugin (struct scan_globals *globals, struct scheduler_plugin *plugin,
       return ERR_HOST_DEAD;
     }
 
+  name = nvticache_get_filename (oid);
+  nasl_set_plugin_filename (name);
```

Comment:
It would be better to call this from exec_nasl_script(), which already has the filename:
- No extra Redis command for every launched plugin.
- Would apply to openvas-nasl-* as well, if needed ?